vite.config.ts

```diff
@@ -19,7 +19,11 @@ export default defineConfig({
   },
   plugins: [
     react(),
-    svgr(),
+    svgr({
+      svgrOptions: {
+        icon: true,
+      },
+    }),
```

Comment:
**note:** this is the actual fix. The [icon](https://react-svgr.com/docs/options/#icon) option is what were relying on prior to the Vite migration, but the Vite migration work missed this detail.